paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/EditPaymentMethodViewInteractor.kt

```diff
@@ -33,15 +35,19 @@ internal interface ModifiableEditPaymentMethodViewInteractor : EditPaymentMethod
         fun create(
             initialPaymentMethod: PaymentMethod,
             removeExecutor: PaymentMethodRemoveOperation,
-            updateExecutor: PaymentMethodUpdateOperation
+            updateExecutor: PaymentMethodUpdateOperation,
+            onRemoved: (PaymentMethod) -> Unit,
```

Comment:
Rather than `onRemoved`, should we just do this as part of `removeExecutor`?

Reply:
Hmm, we’d have to execute the navigate as a side-effect before returning the result. It doesn’t feel super clean to me, but I’ll give it a go.